jekyll/_cci2/configuration-reference.md

```diff
@@ -298,9 +299,9 @@ paths | Y | List | List of directories which should be added to the cache
 key | Y | String | Unique identifier for this cache
 {: class="table table-striped"}
 
-Cache for specific `key` is immutable a cannot be changed. If cache for given `key` is already exist it won't be modified and job execution will proceed to a next step.
+The cache for specific `key` is immutable and cannot be changed once written. NOTE: _If the cache for given `key` is already exists it won't be modified, and job execution will proceed to the next step._
```

Comment:
"The cache for *a* specific `key` is immutable and cannot be changed once written. NOTE: _If the cache for *the* given `key` (*remove 'is'*)already exists, it won't be modified, and job execution will proceed to the next step."